src/sqlite_middleware/database.py

```diff
@@ -15,36 +17,32 @@ def __del__(self):
         except sqlite3.ProgrammingError:
             pass
 
-    
     def select_all(self, cls):
         # Give a class to this function, and it will retrieve all objects with this class from the database
         table_name = f"tbl{cls.__name__}s"
 
         sql = f"SELECT * FROM {table_name}"
 
         all_rows = self.cursor.execute(sql).fetchall()
-        
+
         return all_rows
 
-    
     def select_by_id(self, cls, id):
         # Give a class and an id to this function, and it will retrieve the object with this id from the database
         all_rows = self.select_by_custom_attribute(cls, "id", id)
-        
+
         return all_rows
 
-    
     def select_by_custom_attribute(self, cls, custom_attribute, value):
         # Give a class and a custom attribute to this function, and it will retrieve the objects with this custom attribute from the database
         table_name = f"tbl{cls.__name__}s"
 
         sql = f"SELECT * FROM {table_name} WHERE {custom_attribute}='{value}'"
-        print(sql)
+        # print(sql)
```

Comment:
Hey man, thanks for checking out the repo.

Could you instead remove the line all together please. 
Thanks!

Reply:
Sure! Working on it.